data/mods/Magiclysm/items/ethereal_items.json

```
@@ -278,7 +278,7 @@
     "color": "green",
     "pocket_data": [ { "pocket_type": "CONTAINER", "max_contains_volume": "25 L", "max_contains_weight": "75 kg", "moves": 60 } ],
     "material_thickness": 3,
-    "flags": [ "BELTED", "TRADER_AVOID", "WATERPROOF", "STURDY", "NO_UNWIELD", "ONLY_ONE", "NO_REPAIR", "NO_SALVAGE" ],
+    "flags": [ "BELTED", "TRADER_AVOID", "WATERPROOF", "STURDY", "OVERSIZE", "NO_TAKEOFF", "ONLY_ONE", "NO_REPAIR", "NO_SALVAGE" ],
```

```suggestion
    "flags": [ "BELTED", "TRADER_AVOID", "WATERPROOF", "STURDY", "OVERSIZE", "NO_TAKEOFF", "ONLY_ONE", "NO_REPAIR", "NO_SALVAGE", "ZERO_WEIGHT" ],
```
Looks like this should fix the test failure

---

Oh, i didnt even spot it has 0 weight. Guess it also means i need to fix 0 ml volume tho